ansible/roles/setup_course/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Copy setup-course dockerfile
+  copy:
+    src: Dockerfile
+    dest: "{{ working_dir }}/Dockerfile.setup-course"
```

Comment:
Like the image names, we are using vars to identify the Dockerfile names, so we should do the same here.

Reply:
Yes, I agree. Only thinking about how many times or how often these dockerfile names can be changed or used in an Ansible execution